src/main/java/org/sagebionetworks/web/client/widget/entity/act/EmailMessagePreviewModal.java

```diff
@@ -0,0 +1,31 @@
+package org.sagebionetworks.web.client.widget.entity.act;
+
+import com.google.gwt.user.client.ui.IsWidget;
+import com.google.gwt.user.client.ui.Widget;
+import com.google.inject.Inject;
+
+public class EmailMessagePreviewModal implements EmailMessagePreviewModalView.Presenter, IsWidget {
```

Comment:
Just use org.sagebionetworks.web.client.widget.modal.Dialog with your HTML (or HTMLPanel) widget (constructed in your view)